.travis.yml

```diff
@@ -22,6 +22,15 @@ stages:
 
 jobs:
   include:
+    - stage: Test
+      env:
+        - TEST: Editor Config check
+      language: node_js
```

Comment:
We should either specify a Node.js version here (probably 8) or add a `.nvmrc` file to the repo. Currently nvm defaults to v0.10, which is unfortunate. I think adding an `.nvmrc` file is the better option since we probably want a more recent LTS version of Node.js for "Doc Toc Check".

Reference:

- https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions-using-.nvmrc
- https://github.com/creationix/nvm#nvmrc

Reply:
just saying `lts` as version should work, no need for an nvmrc file IMO

Reply:
I would prefer it in the config file like this: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions

Reply:
Adding `node_js: "lts/*"` should work

Reply:
@chorrell I think it's ok now?

Reply:
Yep